drush/Commands/SiteInstallCommands.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drush\Commands;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Consolidation\AnnotatedCommand\CommandResult;
+use Drupal\cohesion\Drush\DX8CommandHelpers;
+
+/**
+ * Rebuild site studio when site is installed via drush.
+ */
+class SiteInstallCommands extends DrushCommands {
+
+  /**
+   * Do a forceful rebuild whenever site is installed.
+   *
+   * @hook post-command site-install
+   */
+  public function postCommand($result, CommandData $commandData) {
+    $profile = \Drupal::service('config.factory')->get('core.extension')->get('profile');
```

Comment:
@saurabhtripathics - you don't need to get the profile from here. It should be available as part of CommandData.

Reply:
done